nightingale_perception/config/movo_body_filter.yaml

```diff
@@ -0,0 +1,25 @@
+cloud_filter_chain:
+- name: body_filter
+  type: robot_body_filter/RobotBodyFilterPointCloud2
+  params:
+     frames/sensor: "camera_depth_optical_frame"
+     sensor/point_by_point: False
+     sensor/min_distance: 0.05
+     body_model/inflation/scale: 1.07
+     body_model/inflation/padding: 0.03
+     filter/do_shadow_test: False
+     filter/max_shadow_distance: 2.0
+     transforms/buffer_length: 15.0
+     transforms/timeout/reachable: 0.2
+     transforms/timeout/unreachable: 0.2
+     #filter/model_pose_update_interval: 0.005 # arbitrary update interval selected for now
+     only_links: ["left_arm_half_2_link", "left_forearm_link", "left_gripper_base_link", 
```

Comment:
Any reason to include shoulder links here? Will the camera ever see the shoulder? Otherwise I think this list of links is correct

Reply:
The shoulder join has its own link which is not listed here. I wasn't sure if the long joint closest to the shoulder would ever cross the path of the camera but I left it in just to be safe.